Glossary.html

```diff
@@ -4,17 +4,18 @@
         
     <title>Alison's CSD120 Website</title>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initail-scale=1.0">
-    <link rel="stylesheet" href="Styles/mobilemain.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="Styles/normalize.css">
    <link rel="stylesheet" href="Styles/main.css">
+   <link rel="stylesheet" href="Styles/mobilemain.css">
```

Comment:
Typo in viewport meta tag (`initail` vs `initial`)
Your mobile styles need to be loaded AFTER the main styles so they will override the main styles when the media query matches